src/lib/CONFIG/config.h

```diff
@@ -201,7 +201,9 @@ typedef union {
 typedef struct __attribute__((packed)) {
     uint32_t    version;
     uint8_t     uid[UID_LEN];
-    uint8_t     unused_padding[2];
```

Comment:
Shouldn't this change bump config version and add mapping as well _legacy structure?

Reply:
No, as these are unused they will be 0 by default which is NONE.
And this second serial selection is new for the master branch.

Reply:
might have separate question - deep dived into the logic - why serial 0 is having default CRSF and Serial1 has NONE?

Reply:
The addition of a second serial interface allowing to run a second protocol in parallel is a true extension. Nothing was changed to the way serial0 was working. With serial1 initialized to NONE 3.5 exhibits no difference to the way 3.4 and previous versions present a virgin receiver to the user. CRSF is still the default protocol ready to talk to an FC just like the founding fathers intended it (I guess).